### ISSUE TITLE
OP_REQUIRES failed at save_restore_v2_ops.cc:184 : Not found: Key output_bias not found in checkpoint

### ISSUE THREAD

**Issue by Faker0918** [google-research/bert#1278]
I have pre-trained a model based on the bert-large model using my dataset and get the checkpoint successfully, 
however ,when I use the checkpoint to do predict task I get the **Restoring from checkpoint failed. **error
i just run
****
`python run_classifier.py
--task_name=XNLI
\
--do_train=true
\
--do_eval=true
\
--data_dir=./GLUE_DIR/XNLI
\
--vocab_file=./wwm_uncased_L-24_H-1024_A-16/vocab.txt
\
--bert_config_file=./wwm_uncased_L-24_H-1024_A-16/bert_config.json
\
--init_checkpoint=./tmp/pretraining_output/model.ckpt-20
\
--max_seq_length=64
\
--train_batch_size=8
\
--learning_rate=2e-5
\
--num_train_epochs=5.0
\
--output_dir=./tmp/pretraining_output
`
2021-11-24 20:28:22.908571: W tensorflow/core/framework/op_kernel.cc:1651] OP_REQUIRES failed at save_restore_v2_ops.cc:184 : Not found: Key output_bias not found in checkpoint
ERROR:tensorflow:Error recorded from training_loop: Restoring from checkpoint failed. This is most likely due to a Variable name or other graph key that is missing from the checkpoint. Please ensure that you have not altered the graph expected based on the checkpoint. Original error:

Key output_bias not found in checkpoint
	 [[node save/RestoreV2 (defined at /anaconda3/envs/tf_ruan/lib/python3.7/site-packages/tensorflow_core/python/framework/ops.py:1748) ]]

Original stack trace for 'save/RestoreV2':
  File "/xuweixinxi/run_classifier.py", line 1007, in <module>
    tf.app.run()
  File "/anaconda3/envs/tf_ruan/lib/python3.7/site-packages/tensorflow_core/python/platform/app.py", line 40, in run
    _run(main=main, argv=argv, flags_parser=_parse_flags_tolerate_undef)
  File "/anaconda3/envs/tf_ruan/lib/python3.7/site-packages/absl/app.py", line 303, in run
    _run_main(main, args)
  File "/anaconda3/envs/tf_ruan/lib/python3.7/site-packages/absl/app.py", line 251, in _run_main
    sys.exit(main(argv))
  File "/xuweixinxi/run_classifier.py", line 906, in main
    estimator.train(input_fn=train_input_fn, max_steps=num_train_steps)
  File "/anaconda3/envs/tf_ruan/lib/python3.7/site-packages/tensorflow_estimator/python/estimator/tpu/tpu_estimator.py", line 3030, in train
    saving_listeners=saving_listeners)
  File "/anaconda3/envs/tf_ruan/lib/python3.7/site-packages/tensorflow_estimator/python/estimator/estimator.py", line 370, in train
    loss = self._train_model(input_fn, hooks, saving_listeners)
  File "/anaconda3/envs/tf_ruan/lib/python3.7/site-packages/tensorflow_estimator/python/estimator/estimator.py", line 1161, in _train_model
    return self._train_model_default(input_fn, hooks, saving_listeners)
  File "/anaconda3/envs/tf_ruan/lib/python3.7/site-packages/tensorflow_estimator/python/estimator/estimator.py", line 1195, in _train_model_default
    saving_listeners)
  File "/anaconda3/envs/tf_ruan/lib/python3.7/site-packages/tensorflow_estimator/python/estimator/estimator.py", line 1490, in _train_with_estimator_spec
    log_step_count_steps=log_step_count_steps) as mon_sess:
  File "/anaconda3/envs/tf_ruan/lib/python3.7/site-packages/tensorflow_core/python/training/monitored_session.py", line 584, in MonitoredTrainingSession
    stop_grace_period_secs=stop_grace_period_secs)
  File "/anaconda3/envs/tf_ruan/lib/python3.7/site-packages/tensorflow_core/python/training/monitored_session.py", line 1014, in __init__
    stop_grace_period_secs=stop_grace_period_secs)
  File "/anaconda3/envs/tf_ruan/lib/python3.7/site-packages/tensorflow_core/python/training/monitored_session.py", line 725, in __init__
    self._sess = _RecoverableSession(self._coordinated_creator)
  File "/anaconda3/envs/tf_ruan/lib/python3.7/site-packages/tensorflow_core/python/training/monitored_session.py", line 1207, in __init__
    _WrappedSession.__init__(self, self._create_session())
  File "/anaconda3/envs/tf_ruan/lib/python3.7/site-packages/tensorflow_core/python/training/monitored_session.py", line 1212, in _create_session
    return self._sess_creator.create_session()
  File "/anaconda3/envs/tf_ruan/lib/python3.7/site-packages/tensorflow_core/python/training/monitored_session.py", line 878, in create_session
    self.tf_sess = self._session_creator.create_session()
  File "/anaconda3/envs/tf_ruan/lib/python3.7/site-packages/tensorflow_core/python/training/monitored_session.py", line 638, in create_session
    self._scaffold.finalize()
  File "/anaconda3/envs/tf_ruan/lib/python3.7/site-packages/tensorflow_core/python/training/monitored_session.py", line 237, in finalize
    self._saver.build()
  File "/anaconda3/envs/tf_ruan/lib/python3.7/site-packages/tensorflow_core/python/training/saver.py", line 840, in build
    self._build(self._filename, build_save=True, build_restore=True)
  File "/anaconda3/envs/tf_ruan/lib/python3.7/site-packages/tensorflow_core/python/training/saver.py", line 878, in _build
    build_restore=build_restore)
  File "/anaconda3/envs/tf_ruan/lib/python3.7/site-packages/tensorflow_core/python/training/saver.py", line 502, in _build_internal
    restore_sequentially, reshape)
  File "/anaconda3/envs/tf_ruan/lib/python3.7/site-packages/tensorflow_core/python/training/saver.py", line 381, in _AddShardedRestoreOps
    name="restore_shard"))
  File "/anaconda3/envs/tf_ruan/lib/python3.7/site-packages/tensorflow_core/python/training/saver.py", line 328, in _AddRestoreOps
    restore_sequentially)
  File "/anaconda3/envs/tf_ruan/lib/python3.7/site-packages/tensorflow_core/python/training/saver.py", line 575, in bulk_restore
    return io_ops.restore_v2(filename_tensor, names, slices, dtypes)
  File "/anaconda3/envs/tf_ruan/lib/python3.7/site-packages/tensorflow_core/python/ops/gen_io_ops.py", line 1696, in restore_v2
    name=name)
  File "/anaconda3/envs/tf_ruan/lib/python3.7/site-packages/tensorflow_core/python/framework/op_def_library.py", line 794, in _apply_op_helper
    op_def=op_def)
  File "/anaconda3/envs/tf_ruan/lib/python3.7/site-packages/tensorflow_core/python/util/deprecation.py", line 507, in new_func
    return func(*args, **kwargs)
  File "/anaconda3/envs/tf_ruan/lib/python3.7/site-packages/tensorflow_core/python/framework/ops.py", line 3357, in create_op
    attrs, op_def, compute_device)
  File "/anaconda3/envs/tf_ruan/lib/python3.7/site-packages/tensorflow_core/python/framework/ops.py", line 3426, in _create_op_internal
    op_def=op_def)
  File "/anaconda3/envs/tf_ruan/lib/python3.7/site-packages/tensorflow_core/python/framework/ops.py", line 1748, in __init__
    self._traceback = tf_stack.extract_stack()
